#! /usr/bin/env python

PACKAGE='controller'
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#       Name                Type        Level     Description                               Default
gen.add("lambda_x",         double_t,    0,       "Gain for decay rate in X-direction",      0.2)
gen.add("lambda_y",         double_t,    0,       "Gain for decay rate in Y-direction",      0.2)
gen.add("lambda_theta",     double_t,    0,       "Gain for decay rate in \theta",           5.0)
gen.add("lambda_thetaDot",  double_t,    0,       "Gain for \theta_{dot}",                   1)

exit(gen.generate(PACKAGE, "controller", "controllerGainSet"))
